account-kit/smart-contracts/src/ma-v2/client/client.ts

```diff
@@ -64,7 +97,21 @@ export function createSMAV2AccountClient<
  */
 export async function createSMAV2AccountClient(
   config: CreateSMAV2AccountClientParams
-): Promise<SmartAccountClient> {
+): Promise<SmartAccountClient | AlchemySmartAccountClient> {
```

Comment:
for my own understanding: does it complain if this is just `SmartAccountClient`? technically the AlchemySmartAccountClient is a `SmartAccountClient`